app/src/main/res/layout/fragment_monster_list.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.recyclerview.widget.RecyclerView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/monster_list_recyclerview"
+    android:name="com.theuhooi.uhooipicbook.monsterlist.MonsterListFragment"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_marginStart="@dimen/left_margin"
+    android:layout_marginEnd="@dimen/right_margin"
```

Comment:
Use `paddingStart` and `paddingEnd` with `clipToPadding=false`, otherwise bounce effects are also clipped out.

Reply:
I did not know.
Fix it.

Reply:
`clipToPadding=false` doesn't work with `marginStart` and `marginEnd`.

As you can see in following screenshot, the scroll area clipped because margin means outside offset of the view.
![Uploading Screenshot_20200427-230243(1).png…]()

Try to enable scrollbars and it helps you understand!

Reply:
I misread "padding" and "margin"😢
I understand.
Fix it.